app/controllers/discover.js

```diff
@@ -121,6 +127,39 @@ export default ApplicationController.extend({
         });
     },
 
+    transformTypes(obj) {
```

Comment:
The keys need to be lowercased and have a space between the words for the filters to work properly (at least as they are now). 

There is a lot of code that deals with the filters: 
* to and from url query params
* adding and removing from the facet
* pretty much every time a facet changes.